docs/source/examples/update_dataset.rst

```diff
@@ -0,0 +1,57 @@
+#################
+ Update Dataset
+#################
+
+This topic describes how to update dataset, including:
+
+- :ref:`examples/update_dataset:Update Label`
+
+This examples assumes that you already uploaded a dataset to tensorbay.
+If you haven't done so already, See :ref:`features/dataset_management:Upload Dataset`
+
+There is a scenario with update dataset
+
+# Upload dataset and commit as version 1.0.
+# Update label on dataset and commit as version 2.0.
+# Add data to dataset and commit as version 3.0.
+# ...
+
+See the following detailed instructions.
+
+***************
+ Update Label
+***************
+
+TensorBay SDK supports methods to update label. You can upload label to overwrite previous label.
```

Comment:
labels

Please describe features in the third person. 

These two comments above are also applicable for the following part.